tests/framework-tests/preferences/js/SeparatedPanelPrefsEditorTests.js

```diff
@@ -81,6 +81,12 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         var prefsEditor = separatedPanel.prefsEditor;
         jqUnit.assertEquals("Reset button is invisible", false, $(".flc-prefsEditor-reset").is(":visible"));
         fluid.tests.prefs.assertPresent(prefsEditor, fluid.tests.prefs.expectedComponents["fluid.prefs.separatedPanel"]);
+
+        var toggleAriaPressed = separatedPanel.slidingPanel.locate("toggleButton").attr("aria-pressed");
+        var panelAriaExpanded = separatedPanel.locate("iframe").attr("aria-expanded");
```

Comment:
similar to previous comment, a rename might make it more clear what this variable actually holds.